### PR TITLE
add French translations

### PR DESCRIPTION
**Pull request by lucieho**
This adds French translation for all current string resources.

### CODE REVIEW

File: app/src/main/res/values-fr-rFR/strings.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<resources>
+    <string name="app_name">CoronaGame</string>
+    <string name="can_t_Download_Offline">Téléchargement hors ligne impossible</string>
+    <string name="can_t_Upload_Offline">Transfert hors ligne impossible</string>
+    <string name="launch_history">Lancer l\'historique</string>
+    <string name="launch_gps">Lancer le GPS</string>
+    <string name="launch_firebase">Lancer Firebase</string>
+    <string name="i_am_infected">Je suis infecté(e)</string>
+    <string name="i_am_cured">Je ne suis pas infecté(e)</string>
+    <string name="gps_status_on">"Signal GPS acquis "</string>
```
Comment (Ulydev) [inline app/src/main/res/values-fr-rFR/strings.xml:11]: the double quotes here might be a typo

**Review by Ulydev — APPROVED** [review]
Nice feature. We might want to consider moving the localized resources to "values-fr" without specifying the region so that the app may be translated in French-speaking countries other than France

File: app/src/main/res/values-fr-rFR/strings.xml (hunk 1 above)
Comment (lucieho) [reply app/src/main/res/values-fr-rFR/strings.xml:11]: That's a bit surprising indeed, I actually didn't edit this file directly but used the Translations Editor in Android studio. Seems like the quotes don't make a difference

**Comment by lucieho** [comment]
> Nice feature. We might want to consider moving the localized resources to "values-fr" without specifying the region so that the app may be translated in French-speaking countries other than France

From what I understood the usage of values-fr rather than default values depends on the phone's language settings, so there shouldn't be any problem on this part.

**Review by TheCrazyKing — APPROVED** [review]
Ok great idea. However, I would do all the translations at the end at once. And German is still missing :)

**Pull request merged**